lingui.config.ts

```diff
@@ -1,8 +1,63 @@
+import { default as babelExtractor } from '@lingui/cli/api/extractors/babel'
```

Comment:
is this something you could upstream to this package?

Reply:
yes, but it'd need to be a little more fully-featured.
but if we made time for it I'd love to do it.